tests/type-system/extract-pass/test.wake

```diff
@@ -1,9 +1,21 @@
-def Pair (_: Integer) (_: String)                      = Pair 44 "as"
+def Pair (_: Integer) (_: String) = Pair 44 "as"
+
+
 def Pair (_: Integer) (_: String): Pair Integer String = Pair 44 "as"
+
+
 def Pair _ _:                      Pair Integer String = Pair 44 "as"
```

Comment:
This should probably not keep quite so many spaces, especially since we've gotten rid of other places which had been aligned.

Reply:
Ah yeah, this is a bug. Probably an edge case where there currently isn't a rule